doc/api/tracing.md

```diff
@@ -47,4 +46,35 @@ as the one used by `process.hrtime()`
 however the trace-event timestamps are expressed in microseconds,
 unlike `process.hrtime()` which returns nanoseconds.
 
+## The `trace_events` module
+
+The `trace_events` module may be used to turn trace event reporting on and off.
+
+It is accessible using `require('trace_events')`.
+
+### trace_events.setTracingCategories(categories)
+
+* `categories` [string] A comma-separated list of category names. If
+  `categories` is `undefined`, `null`, or an empty string, capture of trace
+  events will be stopped.
+
+Sets the currently enabled trace event category names and enables trace event
+capture.
+
+An error will be thrown if trace event support is not enabled in the Node.js
+binary.
+
+### trace_events.getTracingCategories()
+
+* Return: [string]
+
+Returns the current list of category names as a comma-separated list or
+`undefined`.
```

Comment:
Should we return `undefined`? Wouldn't an empty array be better here?

Reply:
would prefer not to do a useless allocation.